plugins/backup/veeam/src/main/java/org/apache/cloudstack/backup/veeam/VeeamClient.java

```diff
@@ -616,18 +633,20 @@ public boolean deleteJobAndBackup(final String jobName) {
     public List<Backup.RestorePoint> listRestorePoints(String backupName, String vmInternalName) {
         final List<String> cmds = Arrays.asList(
                 String.format("$backup = Get-VBRBackup -Name \"%s\"", backupName),
-                String.format("if ($backup) { (Get-VBRRestorePoint -Backup:$backup -Name \"%s\" ^| Where-Object {$_.IsConsistent -eq $true}) }", vmInternalName)
+                String.format("if ($backup) { $restore = (Get-VBRRestorePoint -Backup:$backup -Name \"%s\" ^| Where-Object {$_.IsConsistent -eq $true})", vmInternalName),
+                "if ($restore) { $restore ^| Format-List } }"
```

Comment:
@SadiJr Is this cmd change tested?

Reply:
Yes, I tested in Veeam B&R v11 (build 11.0.0.837), Enterprise license. Works fine. And, according to the powershell documentation (https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/format-list?view=powershell-7.1), when using Format-List, the commands will be output in the pattern expected by ACS. 

But I don't tested in Veeam 9.x.

Reply:
Code appears should be backward compatible